emm-docs/content/subcommands/git-branch-pull.md

```diff
@@ -0,0 +1,19 @@
+---
+weight: 9
+---
+Use `git-branch-pull` subcommand to get the latest changes for your base repo and sync all 
+modules to the latest commit of the base repo.
+
+{{< hint warning >}}
+**NOTE**
+Since Evergreen builds are only run against a single branch. You will only want to run this
+subcommand on on the branch tracked by Evergreen. 
```

Comment:
[nit] can you remove one of the `on`s here?

Reply:
done.